packages/mongodb-rag-ingest/src/commands/all.ts

```diff
@@ -48,6 +48,17 @@ export const doAllCommand = async (
     since: lastSuccessfulRunDate ?? new Date("2023-01-01"),
   });
 
+  // cleanup - delete pages and embedded content that are no longer in the data sources
+  await config.pageStore.deletePages({
+    dataSources: config.dataSources.map(({ name }) => name),
+    permanent: true,
```

Comment:
should this be a CLI flag?

Reply:
It can be. My understanding was that we wanted to permanently delete all data sources that are not in the list any time we run `ingest all`. If we want the option to mark them for deletion rather than permanently delete, i can add it as a CLI flag

Reply:
perhaps we'd like `ingest:all` and `ingest: all --cleanup` or something like that to indicate we want to ingest all and delete any pages and embeddings that aren't part of the datasource list?

Reply:
as default behavior, please do non-permanent delete. 

`--permanentDeletePages` as flag for permadelete